### DISCUSSION TITLE
I lost my MFA device and unable to login

### DISCUSSION THREAD

**Discussion by software-pulse** [community/community#125172] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hello Github support,

I lost my MFA device and am unable to login github with my credentials. I tried to solve this in multiple ways like github Virtual Assistant, documentation etc. but no use. I don't know how to reach github customer support. Kindly advise how to proceed. This is really urgent. 

Thanks,
Ragav

**Comment by davevad93** [comment]
Hi @software-pulse ,

To contact GitHub support you can create a ticket [here](https://support.github.com/request) or contact them via email: [support@github.com](mailto:support@github.com)

**Reply by software-pulse** [reply]
I am unable to create a ticket. I suppose, because i am not using licensed version. when i tap "New ticket" its redirected to virtual assistant page. It's not useful. Also, when i sent email to support@github.com, it's not delivered. Its says "message blocked". see screen capture.
![Screenshot 2024-05-22 171413](https://github.com/community/community/assets/38241539/a77abd38-0400-4c80-8157-886327924996)

**Reply by davevad93** [reply]
Hey @software-pulse , try opening a ticket [here](https://support.github.com/contact/account) and select one between `Device verification or sign-in unable to access email `, `Two-factor authentication recovery`, `Sign-in, verification or other account questions`, choose the one you think is most relevant to your problem. Here you can specify your issue without being redirected to a virtual assistant.

**Reply by software-pulse** [reply]
Thanks dude. It's working. I raised the ticket. I suppose they will take more time to respond my email (ticket). Still seeking for answer. let's see

**Reply by davevad93** [reply]
You're welcome @software-pulse , I hope you can solve that!
